src/task/MonitorTask.ts

```diff
@@ -262,6 +262,11 @@ export default class MonitorTask
   }
 
   connectionHealthDidChange(connectionHealthData: ConnectionHealthData): void {
+    if (!this.context.meetingSessionConfiguration?.urls?.audioHostURL?.length) {
+      // no audio host information, no need to reconnect due to ping pong
```

Comment:
Is this true? Ping pong is to monitor the connection with the signaling channel (video right)?

Reply:
for keeping the behavior same with other chime  sdk, video connection is not monitored by ping pong method and it relies on audio connection for monitoring

Reply:
Are you saying that there is some backend incompatibility with ping-pong when audio is not used?  Kinda confused here.

Reply:
fixed comment